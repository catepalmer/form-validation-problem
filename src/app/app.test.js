import Enzyme, { shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import toJson from 'enzyme-to-json'
import 'jest-enzyme'
import React from 'react'

import App from './app.component'

Enzyme.configure({ adapter: new Adapter() })

//APP COMPONENT
test('Renders the App component', () => {
  expect(toJson(shallow(<App />))).toMatchSnapshot()
})