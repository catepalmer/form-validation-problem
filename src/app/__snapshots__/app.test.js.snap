// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the App component 1`] = `
<div
  className="container"
>
  <Connect(Form) />
</div>
`;
